core/embed/extmod/modtrezorui/display.c

```diff
@@ -45,6 +46,15 @@
 #include "display_interface.h"
 
 static struct { int x, y; } DISPLAY_OFFSET;
+static uint8_t display_jpeg_work[3100];
```

Comment:
Can we extract `3100` as a constant or a define and describe how this is is calculated?

Reply:
added the comment: https://github.com/trezor/trezor-firmware/commit/c3f19f977e7e5a4c7f683f8789a774c8caca8dfa

Reply:
added the comment: https://github.com/trezor/trezor-firmware/commit/2a3015b5100006d8dd6324fceb840be2b9dba594